### DISCUSSION TITLE
The RowLayout has bad layout effects

### DISCUSSION THREAD

**Discussion by wolfgang-ch** [mytourbook/mytourbook#1019] [General]
Depending on the tag image size or tag content width, the RowLayout can increase the height of the tour description more than neccesarry so that a vertical scrollbar is displayed even when this is not needed because the description height is set only for the minimum height

The GridLayout has a fixed number of columns in this example

Because of this, I'll use the GridLayout with a customizeable number of tag content columns

![image](https://user-images.githubusercontent.com/1283445/224630929-6a9d007b-1a5f-4edc-8ab5-3db626784c26.png)

**Comment by wolfgang-ch** [comment]
The RowLayout has an interresting feature to fill the horizontal space but with this bad layout effect, the GridLayout seems to be more robust for the all the layouts which are within the tour editor.

**Reply by FJBDev** [reply]
Good finding, I thought that RowLayout would be a good fit for that tags but evidently the bad layout effect make it not worth it.

**Reply by wolfgang-ch** [reply]
I've never used RowLayout so I had no experience with it. The RowLayout would be good in this case but in very complex layouts, which the tour editor contains, it fails.
